src/builder/Actions.sol

```diff
@@ -58,6 +58,9 @@ library Actions {
     uint256 constant SWAP_EXPIRY_BUFFER = 3 days;
     uint256 constant TRANSFER_EXPIRY_BUFFER = 7 days;
 
+    /* replay counts */
+    uint256 constant RECURRING_SWAP_REPLAY_COUNT = 500;
```

Comment:
I tend to think 500 is pretty high. We should just discuss as a product question generally

Reply:
Should be easy to change this when we want to.